test/common/stats/thread_local_store_test.cc

```diff
@@ -1149,6 +1149,36 @@ TEST_F(StatsMatcherTLSTest, RejectPrefixNoDot) {
   EXPECT_MEMORY_LE(mem_consumed, 3500000);
 }
 
+TEST_F(StatsMatcherTLSTest, DoNotRejectHiddenRegex) {
+  InSequence s;
+
+  stats_config_.mutable_stats_matcher()->mutable_exclusion_list()->add_patterns()->MergeFrom(
+      TestUtility::createRegexMatcher(".*[A-Z].*"));
```

Comment:
Why such a complex regex? Why not simply '".*"'?

Reply:
Fair enough. Changed

Reply:
it's still complex.

Reply:
When I said "it's still complex", github was showing me a different version. I still haven't totally wrapped my arms around the data model for github reveiws.

This is fine. It's a little subtle that you are intending to create an inclusion list requiring a single upper-case letter, if I'm reading your intent right. Maybe just add a comment?